### ISSUE TITLE
conditional implementation

### ISSUE THREAD

**Issue by PieraRiccio** [rosinality/stylegan2-pytorch#166]
Hi, did you also implement a conditional implementation, so that I can pass files with labels, as in the original TensorFlow implementation?

**Comment by rosinality** [comment]
Currently this implementation does not support label supervision. But it will be not very hard to add it.

**Comment by PieraRiccio** [comment]
Hi @rosinality , thank you for your answer. Okay then, I will try to add it (and let you know if I manage). If in the meanwhile you add it, please keep me updated! :)

**Comment by PieraRiccio** [comment]
Hey, I forked your repository and made a fast (and not very general) implementation yesterday. I am still checking if it converges on my dataset, but maybe you may like to give a look at it. :)

https://github.com/PieraRiccio/stylegan2-pytorch

**Comment by rosinality** [comment]
Nice work! Definitely it looks like how this implementation should have been.

**Comment by foobar8675** [comment]
Hi @PieraRiccio I just took a quick look at your fork, I know it was a quick implementation, but there are hardcoded 4 values because you have 4 labels?

**Comment by snakch** [comment]
@PieraRiccio Sorry for resurrecting this. I'm trying to convert a conditional model trained using Nvidia's implementation (from https://github.com/NVlabs/stylegan2-ada-pytorch ) to rosinality's format.

Am I correct in assuming you followed the same implementation of label conditioning as the NVIDIA implementation? Thank you!

**Comment by PieraRiccio** [comment]
Hey there! The PyTorch implementation for the conditional training did not exist at that time, but I replicated the implementation in TensorFlow, so I guess the answer to this question is: yes! :)
